CONTRIBUTING.md

```diff
@@ -50,11 +50,31 @@ include BREAKING CHANGE keyword for breaking changes.
 
 The message summary should be a one-sentence description of the change, and it must be 72 characters in length or shorter. For a list of tags, please [click here](https://github.com/conventional-changelog/commitlint/tree/master/%40commitlint/config-conventional#type-enum). We use the angular style for tags. See the [default release rules](https://github.com/semantic-release/commit-analyzer/blob/master/lib/default-release-rules.js) based on the commit tag. Shown below is an example of the release type that will be done based on a commit message.
 
-| Commit message                                                                                                                                          | Release type               |
-| ------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------- |
-| `fix(preview): fullscreen behavior in ie11`                                                                                                             | Patch Release              |
-| `feat(preview): add ability to fullscreen`                                                                                                              | ~~Minor~~ Feature Release  |
-| `feat(preview): remove fullscreen ability`<br><br>`BREAKING CHANGE: The fullscreen ability has been removed due to poor support of the fullscreen api.` | ~~Major~~ Breaking Release |
+#### Semantic Versioning
+
+"Semantic release" means that changes to the version number of the package (e.g. `3.42.11` to `3.43.0`) are done according to rules that indicate how the change will affect consumers.
+
+The version number is broken into 3 positions &mdash; `Major.Minor.Patch`. In semantic release terms, changes to the numbers follow `Breaking.Feature.Fix`.
+
```

Comment:
The content below is excellent and brings more color to the commit-types.

I'd like to see it moved under a new "Commit Types" section. I was thrown off for a moment because it currently lives under "Semantic Versioning."

Reply:
Right now it is under 

### Step 4: Push your feature branch to your fork

Just added that semantic release subheading - what do you think about removing it?

Reply:
changed to `Commit Types`